subsys/spm/Kconfig

```diff
@@ -25,12 +25,16 @@ menuconfig IS_SPM
 	select TRUSTED_EXECUTION_SECURE
 
 if IS_SPM
+
```

Comment:
Why was this changed?  Part of a PR you cherry picked for local testing?

Reply:
Yeah this and the other spm change are from the pr that fixed the broken builds for thingys.  You weren’t suppose to look at these 😛